### PR TITLE
feat: :card_file_box: Undo row num, new active listings by contract view

### PR DESCRIPTION
**Pull request by cif**
This PR

- [X] does a DB schema change
  - [X] hasura permissions are updated
  - [ ] the indexer is halted
  - [ ] the corresponding minterop-producer PR is ready
  - [ ] the corresponding minterop-consumer PR is ready
- [ ] changes pubsub payloads
  - [ ] the indexer is halted
  - [ ] the corresponding minterop-producer PR is ready
  - [ ] the corresponding minterop-consumer PR is ready
- [ ] deprecates hasura permissions
  - [ ] frontend won't break
- [ ] creates hasura triggers/actions/events etc.
  - [ ] postgres DB user has been granted appropriate permissions

### CODE REVIEW

**Review by sainthiago — APPROVED** [review]

File: migrations/2023-01-20-103450_undo-row-add-listings-by-contract/down.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-drop view mb_views.nft_metadata_unburned;
+drop view mb_views.active_listings_by_contract;
```
Comment (tifrel) [inline migrations/2023-01-20-103450_undo-row-add-listings-by-contract/down.sql]: Don't we need to remove this file?

File: migrations/2023-01-20-103450_undo-row-add-listings-by-contract/up.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
+-- add listings by contract view
```
Comment (tifrel) [inline migrations/2023-01-20-103450_undo-row-add-listings-by-contract/up.sql]: Don't we need to remove this file?

File: migrations/2023-01-20-103450_add-listings-by-contract/up.sql [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  from nft_listings l
+  where l.nft_contract_id = c.id and l.unlisted_at is null and l.accepted_at is null and l.invalidated_at is null
+) tl on true
+left join lateral (
+  select m.title, m.media -- future! select media type, other ref json fields of interest
+  from nft_metadata m
+  where cl.metadata_id = m.id and c.id = m.nft_contract_id
+  limit 1
+) cmd on true
+where cl.price is not null -- omit null lateral appendages
```
Comment (tifrel) [inline migrations/2023-01-20-103450_add-listings-by-contract/up.sql:43]: missing semicolon

**Review by tifrel — CHANGES_REQUESTED** [review]

File: migrations/2023-01-20-103450_undo-row-add-listings-by-contract/down.sql (hunk 1 above)
Comment (cif) [reply migrations/2023-01-20-103450_undo-row-add-listings-by-contract/down.sql]: 🔥

**Review by tifrel — APPROVED** [review]

**Pull request merged**